rust/src/static/node/mod.rs

```diff
@@ -9,6 +9,8 @@ use serde::Serialize;
 #[derive(Debug, PartialEq)]
 #[cfg_attr(feature = "serde_derive", derive(Serialize))]
 pub struct Node {
+    /// Node code
+    pub code: &'static str,
```

Comment:
static str will increase binary size because it expects that this property to exist throughout the program lifetime thus making it in the bss section of the binary in a readonly memory.

Why not storing this in the heap using `String`?

Reply:
Cuz this api module is behind the "static" feature flag, where the data is read only and static